.github/workflows/ci.yml

```diff
@@ -14,6 +14,9 @@ on:
       - ready_for_review
 
 jobs:
+  black-check:
```

Comment:
I think the format needs to be something like this:

```yaml
black-check:
  name: black
  steps:
    - uses: lgeiger/....
```

Reply:
ok seems like this is not the only thing, I'll look at it later

Reply:
or create a new workflow entirely, `black.yml`

Reply:
We should probably look into https://black.readthedocs.io/en/stable/integrations/github_actions.html

Which is referenced in https://github.com/marketplace/actions/run-black-formatter

Reply:
oh that looks better, just add a yml with that!